jcl/src/java.base/share/classes/java/lang/Class.java

```diff
@@ -5111,5 +5095,73 @@ public boolean isHidden() {
 	Object getClassData() {
 		return classData;
 	}
+
+	private native String[] permittedSubclassesImpl();
 /*[ENDIF] JAVA_SPEC_VERSION >= 15 */
+
+/*[IF JAVA_SPEC_VERSION >= 16]*/
+	/**
+	 * Returns the permitted subclasses related to the calling sealed class as an array
+	 * of Class objects. If the calling class is not a sealed class, is an array class,
+	 * or is primitive, then this method returns null instead.
+	 * The order of any classes returned in the array is unspecified, and any classes
+	 * that cannot be loaded are not included in the returned array. The returned array
+	 * may be empty if there are no permitted subclasses.
+	 * 
+	 * @return array of Class objects if permitted subclasses exist or null if not a sealed class.
+	 * 
+	 * @throws SecurityException if access to any of the classes returned in the array is denied
+	 * 
+	 * @since 15
```

Comment:
Please change this to 16. Even though it doesn't match the current spec, it seems a mistake in the spec since this method is clearly introduced in Java 16.